modules/importing-repositories/procedure-enabling-the-bulk-import-from-github-feature.adoc

```diff
@@ -0,0 +1,41 @@
+[id="enabling-and-giving-access-to-the-bulk-import-feature"]
+= Enabling and giving access to the Bulk Import feature
+You can enable the Bulk Import feature for users and give them the necessary permissions to access it.
+.Prerequisites
+* You have link:{authentication-book-url}#enabling-authentication-with-github[configured GitHub authentication and integration].
+
+.Procedure
+
+. The Bulk Import plugins are installed but disabled by default.
+To enable the `./dynamic-plugins/dist/janus-idp-backstage-plugin-bulk-import-backend-dynamic` and `./dynamic-plugins/dist/janus-idp-backstage-plugin-bulk-import` plugins,
+edit your `dynamic-plugins.yaml` with the following content:
++
+.`dynamic-plugins.yaml` fragment
+[source,yaml]
+----
+plugins:
+  - package: ./dynamic-plugins/dist/janus-idp-backstage-plugin-bulk-import-backend-dynamic
+    disabled: false
+  - package: ./dynamic-plugins/dist/janus-idp-backstage-plugin-bulk-import
+    disabled: false
+----
++
+See link:{installing-and-viewing-dynamic-plugins-url}[{installing-and-viewing-dynamic-plugins-title}].
+
+. Configure the required `bulk.import` RBAC permission for the users who are not administrators as follows:
++
+.`rbac-policy.csv` fragment
+[source,csv,subs="+quotes"]
+----
+p, role:default/bulk-import, bulk.import, use, allow
+p, role:default/bulk-import, catalog-entity.read, read, allow
+p, role:default/bulk-import, catalog.entity.create, create, allow
```

Comment:
`catalog-entity.read` and `catalog.entity.create` are not required as far I know for bulk import. cc @rm3l @debsmita1

Reply:
This was meant to be a full example I copied from my own local config. But you're right, it might be confusing. Let's remove the `p, role:default/bulk-import, catalog-entity.read, read, allow` and `p, role:default/bulk-import, catalog.entity.create, create, allow` lines.